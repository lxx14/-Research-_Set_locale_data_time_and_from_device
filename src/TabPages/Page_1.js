import React, { Component } from 'react';
import { NavigationEvents } from 'react-navigation';

import { Text, View } from 'react-native';

import { styles } from './styles';

class TabPage_1 extends Component {
    constructor(props) {
        super(props);
    }

    render() {
        return (
            <View style={styles.container}>
                <Text>Tab page 1</Text>
            </View>
        )
    }
}

export default TabPage_1;