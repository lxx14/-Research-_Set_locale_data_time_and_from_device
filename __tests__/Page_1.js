import 'react-native';
import React from 'react';
import Page_1 from '../src/TabPages/Page_1';

import renderer from 'react-test-renderer';

test('отображается корректно', () => {
  const tree = renderer.create(
    <Page_1 />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
