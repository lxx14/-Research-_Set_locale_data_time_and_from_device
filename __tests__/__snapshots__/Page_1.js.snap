// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`отображается корректно 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Tab page 1
  </Text>
</View>
`;
